include/gatekeeper_sol.h

```diff
@@ -126,9 +117,6 @@ struct sol_config {
 	 */
 	double             req_channel_bw_mbps;
 
-	/* Parameters to setup the mailbox instance. */
-	unsigned int       mailbox_mem_cache_size;
```

Comment:
The Lua file that contains `struct sol_config` must be updated as well as `lua/sol.lua`.